From 94d7682ad81622e2f3d002fb5a03f4f04f91f453 Mon Sep 17 00:00:00 2001
From: Emil Medve <Emilian.Medve@Freescale.com>
Date: Wed, 8 Jan 2014 04:41:02 -0600
Subject: [PATCH 050/466] fsl_qbman: Fix the checkpatch warning type
 EXPORTED_WORLD_WRITABLE

WARNING:EXPORTED_WORLD_WRITABLE: Exporting world writable files is usually an error. Consider more restrictive permissions.

Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
Change-Id: I61561d1a79f4119d91a45dd8788a819e70705a3d
Reviewed-on: http://git.am.freescale.net:8181/7788
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Haiying Wang <Haiying.Wang@freescale.com>
Reviewed-by: Geoff Thorpe <Geoff.Thorpe@freescale.com>
Reviewed-by: Emilian Medve <Emilian.Medve@freescale.com>
[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/staging/fsl_qbman/qman_config.c |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/staging/fsl_qbman/qman_config.c b/drivers/staging/fsl_qbman/qman_config.c
index 4f7d8a4..d57a45d 100644
--- a/drivers/staging/fsl_qbman/qman_config.c
+++ b/drivers/staging/fsl_qbman/qman_config.c
@@ -1152,15 +1152,15 @@ static ssize_t show_sbec(struct device *dev,
 static DEVICE_ATTR(pfdr_fpc, S_IRUSR, show_pfdr_fpc, NULL);
 static DEVICE_ATTR(pfdr_cfg, S_IRUSR, show_pfdr_cfg, set_pfdr_cfg);
 static DEVICE_ATTR(idle_stat, S_IRUSR, show_idle_stat, NULL);
-static DEVICE_ATTR(ci_rlm_avg, (S_IRUSR|S_IWUGO),
+static DEVICE_ATTR(ci_rlm_avg, (S_IRUSR|S_IWUSR),
 		show_ci_rlm_avg, set_ci_rlm_avg);
 static DEVICE_ATTR(err_isr, S_IRUSR, show_err_isr, NULL);
 static DEVICE_ATTR(sfdr_in_use, S_IRUSR, show_sfdr_in_use, NULL);
 
-static DEVICE_ATTR(dcp0_dlm_avg, (S_IRUSR|S_IWUGO), show_dlm_avg, set_dlm_avg);
-static DEVICE_ATTR(dcp1_dlm_avg, (S_IRUSR|S_IWUGO), show_dlm_avg, set_dlm_avg);
-static DEVICE_ATTR(dcp2_dlm_avg, (S_IRUSR|S_IWUGO), show_dlm_avg, set_dlm_avg);
-static DEVICE_ATTR(dcp3_dlm_avg, (S_IRUSR|S_IWUGO), show_dlm_avg, set_dlm_avg);
+static DEVICE_ATTR(dcp0_dlm_avg, (S_IRUSR|S_IWUSR), show_dlm_avg, set_dlm_avg);
+static DEVICE_ATTR(dcp1_dlm_avg, (S_IRUSR|S_IWUSR), show_dlm_avg, set_dlm_avg);
+static DEVICE_ATTR(dcp2_dlm_avg, (S_IRUSR|S_IWUSR), show_dlm_avg, set_dlm_avg);
+static DEVICE_ATTR(dcp3_dlm_avg, (S_IRUSR|S_IWUSR), show_dlm_avg, set_dlm_avg);
 
 static DEVICE_ATTR(sbec_0, S_IRUSR, show_sbec, NULL);
 static DEVICE_ATTR(sbec_1, S_IRUSR, show_sbec, NULL);
-- 
1.7.10.4

